components/library/gamin/gamin-39.p5m

```diff
@@ -11,17 +11,18 @@
 
 #
 # Copyright 2020 Alexander Pyhalov
+# Copyright 2023 Klaus Ziegler
 #
 
-set name=pkg.fmri value=pkg:/library/python/gamin-27@$(IPS_COMPONENT_VERSION),$(BUILD_VERSION)
-set name=pkg.summary value="$(COMPONENT_SUMMARY)"
-set name=info.classification value="org.opensolaris.category.2008:Development/Python"
+set name=pkg.fmri value=pkg:/library/python/gamin-$(PYV)@$(IPS_COMPONENT_VERSION),$(BUILD_VERSION)
```

Comment:
I think it would be better to drop Python version (-39) from the file name, so we do not need to rename this file with every Python upgrade.